ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/TransactionReceipt.java

```diff
@@ -171,26 +173,37 @@ public void writeToWithRevertReason(final RLPOutput out) {
     writeTo(out, true);
   }
 
```

Comment:
We previously tested only `FRONTIER` receipts so roundtripping issues that were exposed are fixed here.